cmd/rename.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/Scalingo/cli/appdetect"
+	"github.com/Scalingo/cli/apps"
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/urfave/cli"
+)
+
+var (
+	RenameCommand = cli.Command{
+		Name:     "rename",
+		Category: "Global",
```

Comment:
Shouldn't it be `App Management`?

Reply:
True, same for `destroy`, updated in next commit